adobe/spectrum-web-components
# fix: add Grid pattern

## Description
This can either target `main` or target `westbrook/express` in order to make it available to the appropriate teams in the next release, please let me know if you have druthers. Being in the `tools` directory prevents this from being listed in our documentation site for now.

## Related issue(s)
- fixes #1969 

## How has this been tested?

    1. Go [here](https://grid--spectrum-web-components.netlify.app/storybook/?path=/story/grid--default)
    2. `Tab` into the Textfield
    3. `Tab` into the Grid
    4. `Tab` into the Textfield AFTER the grid
    5. `Shift + Tab` back into the grid
    6. See that you are focused on the FIRST of all of the elements
    7. Use arrow keys to navigate around the grid
    1. Go [here](https://grid--spectrum-web-components.netlify.app/storybook/?path=/story/grid--default)
    2. Click items in the grid
    3. See that they are shown in the Action Bar
    4. Scroll down a good bit, select some other things
    5. See that they are shown in the Action Bar
    6. Scroll back up
    7. See that your previous selections are maintained

## Types of changes


## Checklist
